src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -0,0 +1,175 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn best practices for good user experiences with the new autoplay policies in Chrome.
+
+{# wf_updated_on: 2017-09-13 #}
+{# wf_published_on: 2017-09-13 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: Learn best practices for good user experiences with the new autoplay policies in Chrome. #}
+{# wf_blink_components: Blink>Media #}
+
+# Autoplay Policy Changes {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's autoplay policies are about to change in 2018 and I'm here to tell
+you why and how this is going to affect video playback with sound. Spoiler
+alert: Users are going to love it!
+
+<figure>
+  <a href="https://imgflip.com/i/ngd6c">
+    <img height="241px" src="https://i.imgflip.com/ngd6c.jpg"
+         title="I will find you and I will pause you"/>
+  </a>
+  <a href="https://imgur.com/9pj5LGN">
+    <img height="241px" src="http://i.imgur.com/9pj5LGN.jpg"
+         title="I don't remember asking you to autoplay a goddamn thing"/>
+  </a>
+  <a href="https://imgur.com/a/p1ZjC">
+    <img height="241px" src="https://i.imgur.com/aFO9wAml.jpg"
+         title="One does not simply autoplay videos"/>
+  </a>
+  <figcaption>
+    <b>Figure 1.</b>
+    Internet memes tagged "autoplay" found on
+    <a href="https://imgflip.com">Imgflip</a> and
+    <a href="https://imgur.com">Imgur</a>
+  </figcaption>
+</figure>
+
+## New Behaviors {: #new-behaviors }
+
+As you may have [noticed], web browsers are moving towards stricter autoplay
+policies in order to improve the web experience for users, minimize the
+incentives to install extensions that block ads, and reduce data consumption on
+expensive and/or constrained network.
+
+With these new autoplay policies, the Chrome team aims to provide a greater
+control to users over content playing in their browser. It will also benefit
+publishers who have legitimate autoplay use cases.
+
+Chrome's autoplay policies are simple:
+
+- Muted autoplay is always allowed.
+- Autoplay with sound is allowed if any of the following conditions are met:
+    - User has interacted with the site (click, tap, etc.)
+    - [Media Engagement Index](#mei) threshold is crossed (desktop only)
+    - Site has been installed using the ["Add to Homescreen" flow] (mobile only)
+- Top frame can [delegate autoplay permission](#iframe) to their iframes to
+  allow autoplay with sound.
+
+### Media Engagement Index (MEI) {: #mei }
```

Comment:
I've never heard of this. Is there a longer article we can link to?

Reply:
It is brand new. I'll link to the design doc.